postmanlabs/openapi-to-postman
# Fix/value prop in example

The OAS spec has this example definition

https://swagger.io/specification/#example-object

So the value property could be used to store the actual value of the example, but when the example object itself has this property, the library takes only that property as the value example so in a definition like this: 

```
schema:
                type: object
                properties:
                  id:
                    type: integer
                    format: int64
                  value:
                    type: string
              example:
                id: "5789-6378-6372-6372"
                value: QA
```
The conversion process generates the body in this way:

QA

Taking only the value of the value property.

The correct value should be:
```
{
  "id": "5789-6378-6372-6372",
  "value": "QA"
}
```

For solving this, we compare the example's value property and the schema, if they do not have the same properties we are going to use the whole example object, otherwise we use the value property as before.
